charts/uplink/tests/defaults_test.yaml

```diff
@@ -0,0 +1,19 @@
+suite: test default behavior of secret.yaml with default values
+templates:
+  - secret.yaml
+tests:
+  - it: should render with default values without error
+    asserts:
+      - isKind:
+          of: Secret
+      - equal:
+          path: metadata.name
+          value: RELEASE-NAME-uplink
+      - notExists:
```

Comment:
I got an error running the test locally from this PR branch :

```
helm unittest --helm3 uplink       
 FAIL   uplink/tests/defaults_test.yaml
        - Execution Error: 
                Assertion type `notExists` is invalid


### Error:  Assertion type `notExists` is invalid


Charts:      1 failed, 1 errored, 0 passed, 1 total
Test Suites: 1 failed, 1 errored, 0 passed, 1 total
Tests:       0 passed, 0 total
Snapshot:    0 passed, 0 total
Time:        1.064083ms

Error: plugin "unittest" exited with error
```

Reply:
Can you check you have the latest 0.3.4 unittest plugin installed?

```
helm plugin ls | grep unit
unittest        0.3.4           Unit test for helm chart in YAML with ease to keep your chart functional and robust.
```

Reply:
```
helm plugin ls | grep unit
unittest        0.2.8   Unit test for helm chart in YAML with ease to keep your chart functional and robust.
```

Reply:
after a `helm plugin update unittest`
I got 
```
helm unittest uplink        

### Chart [ uplink ] uplink

 PASS  test secret.yaml uplink/tests/custom_values_test.yaml
 PASS  test default behavior of secret.yaml with default values uplink/tests/defaults_test.yaml

Charts:      1 passed, 1 total
Test Suites: 2 passed, 2 total
Tests:       2 passed, 2 total
Snapshot:    0 passed, 0 total
Time:        3.526209ms
```

Reply:
you need to update. Be careful, as this plugins changed their GitHub repository (it was a fork from quintus, merged back to helm unit organization).

Start with 

```
helm plugin uninstall unittest
```

Then follow instruction at https://github.com/helm-unittest/helm-unittest to get the 0.3.5

Reply:
> after a `helm plugin update unittest` I got
> 
> ```
> helm unittest uplink        
> 
> ### Chart [ uplink ] uplink
> 
>  PASS  test secret.yaml uplink/tests/custom_values_test.yaml
>  PASS  test default behavior of secret.yaml with default values uplink/tests/defaults_test.yaml
> 
> Charts:      1 passed, 1 total
> Test Suites: 2 passed, 2 total
> Tests:       2 passed, 2 total
> Snapshot:    0 passed, 0 total
> Time:        3.526209ms
> ```

Nice job!